storage/src/mmr/bitmap.rs

```diff
@@ -54,6 +64,9 @@ pub struct Bitmap<H: CHasher, const N: usize> {
     /// update overhead for elements being appended or updated near the tip compared to a more
     /// typical balanced Merkle tree.
     mmr: Mmr<H>,
+
+    /// The number of bitmap chunks that have been pruned.
+    pruned_chunks: usize,
```

Comment:
nit: is this overkill to make a `u64` or is there some reason you used `usize` (maybe consistentcy with elsewhere?)

Reply:
Made it usize only because it's an index into a deque so can be used without conversion.

Reply:
I suggest adding the same warning we added to `MMR` for using this on a 32-but machine maybe?

Reply:
Added warning in the comments for next_bit (which is u64):

```
// Warning: Even though we use u64 identifiers for bits, on 32-bit machines, the maximum
/// addressable bit is limited to (u32::MAX * N * 8).
```